src/org/thoughtcrime/securesms/PassphraseRequiredActionBarActivity.java

```diff
@@ -13,7 +13,6 @@ public abstract class PassphraseRequiredActionBarActivity extends BaseActionBarA
   @Override
   protected final void onCreate(Bundle savedInstanceState) {
     Log.w(TAG, "onCreate(" + savedInstanceState + ")");
-    onPreCreate();
```

Comment:
ftr, there is a logic change for `onPreCreate()` in the startup code:

- before this PR, `onPreCreate()` was called always, unconditionally at the beginning of creation
- with this PR, `onPreCreate()` is _not_ called in case the app is not configured, in case it is configured, it is called _soon_